src/wallet/rpcdump.cpp

```diff
@@ -172,7 +172,13 @@ UniValue importprivkey(const JSONRPCRequest& request)
         }
     }
     if (fRescan) {
-        pwallet->RescanFromTime(TIMESTAMP_MIN, reserver, true /* update */);
+        int64_t scanned_time = pwallet->RescanFromTime(TIMESTAMP_MIN, reserver, true /* update */);
```

Comment:
This code is repeated 4 times, maybe it's worth moving it to a function?

Reply:
In commit "Give an error when rescan is aborted by the user"

> This code is repeated 4 times, maybe it's worth moving it to a function?

Could be nice, though to be honest a lot of other code is duplicated between the import* RPCs too. So effort might be better spent deduping larger chunks or deprecating older RPCs in favor of importmulti.

Reply:
I think that should be part of a separate refactor for deduplicating all of the import* RPCs.

Reply:
Sure.